assets/app/view/game/buy_company_from_other_player.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module View
+  module Game
+    class BuyCompanyFromOtherPlayer < Snabberb::Component
+      include Actionable
+
+      needs :game, store: true
+      needs :selected_companies, default: [], store: true
+      needs :selected_company, default: nil, store: true
+
+      def render
+        @step = @game.round.active_step
+        @current_actions = @step.current_actions
+        return h(:div, {}, '') if !@step.can_buy_any?(@game.current_entity) || !@current_actions.include?('buy_company')
```

Comment:
can just do h(:div) i think? or maybe h(:div, '')

Reply:
Need for this removed in later revision